dataprofiler/tests/profilers/test_utils.py

```diff
@@ -44,3 +44,42 @@ def test_uneven_chunk_sample(self):
             num_chunks += 1
         self.assertEqual(num_chunks, 100 // 7 + 1)
         self.assertCountEqual(all_values, list(range(100)))
+        
+    def test_find_diff(self):
+        """
+        Checks to see if the find difference function is operating
+        appropriately.
+        """
+
+        # Ensure lists and sets are handled appropriately
+        self.assertEqual("unchanged",
+                         utils.find_diff_of_lists_and_sets([3, 2], [2, 3, 2]))
+        self.assertEqual([[1], [2, 3], [4]],
+                         utils.find_diff_of_lists_and_sets([1, 2, 3], [2, 3, 4]))
+        self.assertEqual("unchanged",
+                         utils.find_diff_of_lists_and_sets({3, 2}, {2, 3}))
+        self.assertEqual([[1], [2, 3], [4]],
+                         utils.find_diff_of_lists_and_sets({1, 2, 3}, {2, 3, 4}))
+        self.assertEqual("unchanged",
+                         utils.find_diff_of_lists_and_sets({2, 3}, [2, 3]))
+        self.assertEqual([[1], [2, 3], [4]],
+                         utils.find_diff_of_lists_and_sets([1, 2, 3], {2, 3, 4}))
+        self.assertEqual([None, {1, 2}],
+                         utils.find_diff_of_lists_and_sets(None, {1, 2}))
+
+        # Ensure ints and floats are handled appropriately
+        self.assertEqual(1, utils.find_diff_of_numbers(5, 4))
+        self.assertEqual(1.0, utils.find_diff_of_numbers(5.0, 4.0))
+        self.assertEqual(1.0, utils.find_diff_of_numbers(5.0, 4))
+        self.assertEqual("unchanged", utils.find_diff_of_numbers(5.0, 5.0))
+        self.assertEqual("unchanged", utils.find_diff_of_numbers(5, 5.0))
+        self.assertEqual([4, None],
+                         utils.find_diff_of_numbers(4, None))
+
+        # Ensure strings are handled appropriately
+        self.assertEqual("unchanged",
+                         utils.find_diff_of_strings("Hello", "Hello"))
+        self.assertEqual(["Hello", "team"],
```

Comment:
should the output include three parts `[['Hello'], [], ['team']]`?

Reply:
I dont think so. I'm imagining using this on Data Label for example so, I think the user would want something like:
["PHONE_NUMBER", "BAN"] if the data labels were different